packages/contracts/test/voting/token-voting.ts

```diff
@@ -107,6 +118,26 @@ describe('TokenVoting', function () {
     );
   });
 
+  async function setBalances(balances: {receiver: string; amount: number}[]) {
+    const promises = balances.map(balance =>
+      governanceErc20Mock.setBalance(balance.receiver, balance.amount)
+    );
+    await Promise.all(promises);
+  }
+
+  async function setTotalSupply(totalSupply: number) {
```

Comment:
I think that this function should either throw an error when the passed `totalSupply` is below the current or start burning tokens. 
In tests when I set something I expect it to be that value afterwards

Reply:
It throws `Error: value out-of-bounds (argument="amount", value={"type":"BigNumber","hex":"-0x01"}, code=INVALID_ARGUMENT, version=abi/5.6.0)`

Reply:
Should I throw an explicit error?